sfge/src/main/java/com/salesforce/graph/vertex/UserClassVertex.java

```diff
@@ -40,6 +40,21 @@ public void afterVisit(SymbolProviderVertexVisitor visitor) {
         visitor.afterVisit(this);
     }
 
+    /**
+     * TODO: Check whether this class can be safely made to extend {@link FieldWithModifierVertex}.
```

Comment:
Long-term, it's probably best if UserClassVertex extends FieldWithModifierVertex, but I was too cowardly to attempt it as part of this PR.